test/lib/ufe/testAttribute.py

```diff
@@ -160,8 +176,17 @@ def runUndoRedoUsingMayaSetAttr(self, attr, newVal, decimalPlaces=None):
         setAttrPath = self.getMayaAttrStr(attr)
         if isinstance(newVal, (ufe.Vector3i, ufe.Vector3f, ufe.Vector3d)):
             cmds.setAttr(setAttrPath, newVal.x(), newVal.y(), newVal.z())
+        elif isinstance(newVal, ufe.Vector2f):
```

Comment:
We could support all vector types with a single:
     cmds.setAttr(setAttrPath, *newVal.vector)
If we want to sinplify this a little.